src/components/Categories.js

```diff
@@ -1,21 +1,24 @@
-import React from 'react';
+/* eslint-disable */
+
+import { useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { checkStatus } from '../redux/categories/categories';
 
-const Categories = () => {
-  const status = useSelector((state) => state.categories);
+function Categories() {
```

Comment:
- [ **OPTIONAL** ] For best practices reasons, I think it would be better if you use arrow functions instead of normal functions. This will make your code be refactored and look professional